### PR TITLE
feat: drop CSGO support

### PR DESCRIPTION
**Pull request by Rushaway**
Untested - Compile without errors.

### CODE REVIEW

File: src/addons/sourcemod/scripting/zr/weapons/zmarket.inc [hunk 1]
```diff
@@ -799,7 +799,7 @@ void ZMarketMenuTypeWeapons(int client)
         int weaponindex = GetArrayCell(arrayTypeWeapons, x);
 
         // Get name of weapon.
-        WeaponsGetName(weaponindex, typeweapon, sizeof(typeweapon), (g_Game == Game_CSGO));
```
Comment (Rushaway) [inline src/addons/sourcemod/scripting/zr/weapons/zmarket.inc:802]: I'm not sure about this @maxime1907

File: src/addons/sourcemod/scripting/zr/weapons/menu_weapons.inc [hunk 2]
```diff
@@ -241,7 +241,7 @@ void WeaponsMenuTypeWeapons(int client)
         int weaponindex = GetArrayCell(arrayTypeWeapons, x);
 
         // Get name of weapon.
-        WeaponsGetName(weaponindex, typeweapon, sizeof(typeweapon), (g_Game == Game_CSGO));
```
Comment (maxime1907) [inline src/addons/sourcemod/scripting/zr/weapons/menu_weapons.inc:244]: false

File: src/addons/sourcemod/scripting/zr/weapons/weapons.inc [hunk 3]
```diff
@@ -478,7 +475,7 @@ stock int WeaponsEntityToDisplay(const char[] entityname, char[] display, int di
         }
 
         // The entity names match, so return display.
-        WeaponsGetName(x, display, displaymaxlen, (g_Game == Game_CSGO));
```
Comment (maxime1907) [inline src/addons/sourcemod/scripting/zr/weapons/weapons.inc:481]: false

File: src/addons/sourcemod/scripting/zr/weapons/zmarket.inc [hunk 4]
```diff
@@ -605,7 +605,7 @@ void ZMarketMenuLoadoutWeapons(int client, WeaponsSlot slot)
         if(wslot != slot)
             continue;
 
-        WeaponsGetName(weaponindex, weaponname, sizeof(weaponname), (g_Game == Game_CSGO));
```
Comment (maxime1907) [inline src/addons/sourcemod/scripting/zr/weapons/zmarket.inc:608]: false by default so remove argument completely

File: src/addons/sourcemod/scripting/zr/weapons/zmarket.inc [hunk 5]
```diff
@@ -1039,7 +1039,7 @@ stock bool ZMarketEquip(int client, const char[] weapon, bool rebuy = false)
 
     // Get the display name for the weapon.
     char weapondisplay[WEAPONS_MAX_LENGTH];
-    WeaponsGetName(weaponindex, weapondisplay, sizeof(weapondisplay), (g_Game == Game_CSGO));
```
Comment (maxime1907) [inline src/addons/sourcemod/scripting/zr/weapons/zmarket.inc:1042]: false by default so remove argument completely

**Comment by Rushaway** [comment]
LGTM

**Pull request merged**